.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.3/apache-maven-3.3.3-bin.zip
```

Comment:
It's intended to be project specific, so there's a fixed value of a known version of Maven that works with the particular build in question. If you wanted a URL that is to be used across an organization that would have to be added. Right now it's only meant to be setup on a per project basis.

We can also set additional project specific settings like JVM memory settings, extensions to use and default maven flags to use.